controllers/user.js

```diff
@@ -8,6 +8,7 @@ const HBSHelpers = require("../hbs-helpers");
 const UNSUB_REASONS = require("../unsubscribe_reasons");
 const sha1 = require("../sha1-utils");
 const TIPS = require("../tips");
+const crypto = require("crypto");
```

Comment:
Can we move this require to the top above `isemail`?  module requirements in a certain order help scan-ability.

1. core node modules (e.g., `crypto`)
2. 3rd-party node modules (e.g., `isemail`)
3. modules from elsewhere in the repo (e.g., `../email-utils`)
4. modules from the same directory (e.g., `./module`)